### DISCUSSION TITLE
Are compressed images supported on BT81x platforms?

### DISCUSSION THREAD

**Discussion by jwestes** [RudolphRiedel/FT800-FT813#66] [Q&A] [answered]
I used EAB 2.7.1 to convert a 500x236 transparent PNG to compressed ARGB2 targeting BT81x called BoatOutline.  I then took the bin file and converted that to c and added it to my project.

In my init function, I have the following line to inflate the image:  (MEM_BOAT_OUTLINE = 0x000f8000)
    EVE_cmd_inflate(MEM_BOAT_OUTLINE, BoatOutline, sizeof(BoatOutline));

In my InitStaticContent function I have the following:
    EVE_cmd_dl(DL_COLOR_RGB | WHITE);
    EVE_cmd_dl(DL_BEGIN | EVE_BITMAPS);
    EVE_cmd_setbitmap(MEM_BOAT_OUTLINE, EVE_ARGB2, 500, 236);
    EVE_cmd_dl(VERTEX2F((EVE_HSIZE - 500) / 2, ((EVE_VSIZE - 236) / 2) + 27));
    EVE_cmd_dl(DL_END);

The display shows roughly the top 25% of the image with some slight scrambling at the bottom of the part that is correct and then just a white box under that, 500 wide by about 175 high.

![PXL_20221210_045420786](https://user-images.githubusercontent.com/8738822/206830239-36429f97-c290-452a-ae3e-b3d5a8cd5687.jpg)

**Comment by jwestes** [comment, marked as answer]
I was not leaving enough room for the inflated image, so the static content was overwriting the image content.

![PXL_20221210_060315912](https://user-images.githubusercontent.com/8738822/206834086-4e61a548-76a9-43d6-b5f6-d0841b1ca528.jpg)

**Comment by RudolphRiedel** [comment]
I highly recommend to use ASTC compression for BT81x, ATSC 8x8 for example only uses 2 bits per pixel.

**Reply by jwestes** [reply]
I need something that supports an alpha channel for the boat outline.  Is there still a better choice over ARGB2?

**Reply by RudolphRiedel** [reply]
ASTC is full color with alpha-channel, the full name of the EVE format is for example: COMPRESSED_RGBA_ASTC_8x8_KHR
But as it is a lossy compression for textures it only needs from 8 bit (ASTC_4x4) down to 0.89 bit (ASTC_12x12) per pixel.
Of course ASTC_12x12 has noticeable artifacts but ASTC_8x8 with just two bit per pixel is fine.

**Reply by jwestes** [reply]
The ASTC_8x8 looks great.  Thanks for the tip!
